se/se_epub_lint.py

```diff
@@ -1420,6 +1421,11 @@ def _lint_special_file_checks(self, filename: Path, dom: se.easy_xml.EasyXmlTree
 		if ebook_flags["has_multiple_page_scans"] and not dom.xpath("/html/body//a[contains(@href, '#page-scans')]"):
 			messages.append(LintMessage("m-075", "Multiple page scans found in metadata, but no link to [text]EBOOK_URL#page-scans[/].", se.MESSAGE_TYPE_ERROR, filename))
 
+		# Check for dates without a time wrapper
+		matches = regex.findall(r"\s\d{3,4}\s", file_contents)
+		if matches:
+			messages.append(LintMessage("s-105", "Dates in the colophon need to be wrapped in an [xhtml]<time>[/] element.", se.MESSAGE_TYPE_ERROR, filename))
```

Comment:
It might be possible to exclude BCE dates using xpath, so that a lint exception is not required. The xpath would search for a text node ending in a 3-4 digit string followed by a space (e.g. `\b[0-9]{3,4}\s$`), which in turn is followed by a sibling `<abbr epub:type="se:era">` element matching the regex `^BCE?$`.

Reply:
Also, we should use xpath for this regardless, because this will emit a lint error if there is for example `<abbr epub:type="z3998:given-name">` in the colophon, which is a very common case.